src/new_versions.jl

```diff
@@ -79,6 +82,18 @@ function make_pr_for_new_version(api::GitHub.GitHubAPI,
               latest_version,
               name,
               dep)
+    elseif !bump_compat_containing_equality_specifier &&
+            !isnothing(current_compat_entry_verbatim) &&
+            contains(current_compat_entry_verbatim, '=') &&
+            !contains(current_compat_entry_verbatim, '>') &&
+            !contains(current_compat_entry_verbatim, '<')
+        # To check for an equality specifier (but not an inequality specifier) we look for an equals sign without any
+        # symbols used for an inequality specifier that would also include an equals sign. Namely, greater than and
+        # less than. Other specifiers containing symbols like ≥ shouldn't parse as containing an equals sign.
+        @info("skipping compat entry because it contains an equality specifier",
+            current_compat_entry_verbatim,
```

Comment:
@DilumAluthge I'm not sure if `current_compat_entry_verbatim` will contain the entire compat entry line (e.g., `FFTW = "=1.2.2"`). If that is the case, this needs to be changed.